### ISSUE TITLE
Block syscalls changing file timestamps

### ISSUE THREAD

**Issue by p-e-w** [p-e-w/maybe#16]
Namely:

* `utime`, `utimes`
* `futimes`, `lutimes`
* `utimensat`, `futimens`

**Comment by swapnilsm** [comment]
Have you started working on this?

**Comment by Ruben62** [comment]
No, I don't want to compromise my phone limitations?


Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: "Swapnil S. Mahajan" <notifications@github.com> Date: 5/24/16  12:21 PM  (GMT-08:00) To: p-e-w/maybe <maybe@noreply.github.com> Cc: Ruben62 <roob95127sjca@gmail.com>, Manual <manual@noreply.github.com> Subject: Re: [p-e-w/maybe] Block all syscalls that might permanently modify
  the system (#16) 
Have you started working on this?

**Comment by p-e-w** [comment]
@swapnilsm: Not yet, I'm still searching for a reliable way to identify those syscalls among the hundreds that Linux has in total.

**Comment by swapnilsm** [comment]
Yeah and I couldn't find any good source where system calls have been tagged as read-only or not.
